src/assets/css/footer.scss

```diff
@@ -0,0 +1,206 @@
+//Prueba para tener contenido y ver el footer al final
+body > p {
+  background-color: $moramia-blue;
+  height: 100vw;
+}
+* {
+  box-sizing: border-box;
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  text-decoration: none;
+}
+.footer {
+  background-color: $footer-background;
+  bottom: 0;
+  color: $moramia-white;
+  display: block;
+  left: 0;
+  overflow: hidden;
+  padding: 5em 0;
+  position: relative;
+  transition: all 0.4s ease;
+  width: 100vw;
+}
+.footer-content {
+  display: block;
+  font-family: $font-stack;
+  height: 100%;
+  margin: auto auto;
+  padding: 3em;
+  .footer-cp {
+    color: $cp-color;
+    font-size: 0.8em;
```

Comment:
is this the font-size specified in the ticket?

Reply:
In the ticket it was only specify the font-size of the links, therefore I used 0.8em because it was the one that looks the same size of the prototype site